exercises/rail-fence-cipher/.meta/version.txt

```diff
@@ -0,0 +1 @@
+1.2.40
```

Comment:
Is this correct? 40 seems awfully high :laughing: 

I think the file should also just be called `version` without the `.txt`

Reply:
Oh damn, it's not the Kotlin version here ? :3